pysat/instruments/sw_kp.py

```diff
@@ -302,8 +302,9 @@ def download(date_array, tag, sat_id, data_path, user=None, password=None):
                 try:
                     logger.info('Downloading file for '+date.strftime('%b %Y'))
                     sys.stdout.flush()
-                    ftp.retrbinary('RETR '+fname, open(saved_fname, 'wb').write)
-                    dnames.append(fname)
+                    with TemporaryFileLock(fname + '.Lock', pysat.file_timeout) as tfl:
```

Comment:
Should all the file writes use the NetworkLock to ensure the data 'written' is actually synced to the filesystem?